spec.html

```diff
@@ -25062,7 +25062,7 @@ <h1>parseInt ( _string_, _radix_ )</h1>
           1. If the length of _S_ is at least 2 and the first two code units of _S_ are either *"0x"* or *"0X"*, then
             1. Remove the first two code units from _S_.
             1. Set _R_ to 16.
-        1. If _S_ contains a code unit that is not a radix-_R_ digit, let _Z_ be the substring of _S_ consisting of all code units before the first such code unit; otherwise, let _Z_ be _S_.
+        1. If _S_ contains a code unit that is not a radix-_R_ digit, let _Z_ be the String value equal to the substring of _S_ consisting of all code units before the first such code unit; otherwise, let _Z_ be _S_.
```

Comment:
Why is this phrasing preferable? I like it how it was.

Reply:
Mostly because "the String value equal to the substring of <var>x</var> consisting of…" is the predominant phrasing in the spec, but a little bit because it is more correct. Alternatively, we could define "<dfn>the substring of</dfn> <var>S</var>..." similarly to how [string-concatenation](https://tc39.es/ecma262/#sec-ecmascript-language-types-string-type) is defined and then use the shorter phrasing.

Reply:
I would prefer that, yes.

Reply:
Done; introduced as string-slice for targeted auto-linking.